bciers/apps/reporting/src/data/jsonSchema/newEntrantInformation.tsx

```diff
@@ -0,0 +1,302 @@
+import { RJSFSchema, UiSchema, WidgetProps } from "@rjsf/utils";
+
+import { Typography } from "@mui/material";
+import {
+  ArrayFieldTemplate,
+  FieldTemplate,
+  TitleOnlyFieldTemplate,
+} from "@bciers/components/form/fields";
+import checkboxWidget from "@bciers/components/form/widgets/CheckboxWidget";
+import { DateWidget } from "@bciers/components/form/widgets";
+import { ReadOnlyWidget } from "@bciers/components/form/widgets/readOnly";
+import React from "react";
+import ObjectFieldTemplate from "@rjsf/core/lib/components/templates/ObjectFieldTemplate";
+
+interface FieldTemplateProps {
+  id: string;
+  classNames: string;
+  children: React.ReactNode;
+  formContext: any;
+}
+export const ProductionDataTitleWidget: React.FC<WidgetProps> = ({
+  id,
+  value,
+}) => {
+  return (
+    <div id={id} className="w-full mt-8">
+      <h2 className="inline-block p-0 text-lg font-bold text-bc-bg-blue m-0 mb-12">
+        <u>Product:</u> {value}
+      </h2>
+    </div>
+  );
+};
```

Comment:
That widget is copied from the production data form, we should refactor it in a place where it makes sense. maybe just a `common.tsx` file in that folder, or something like `reporting/src/app/utils/widgets/ProductDataTitleWidget.tsx`

Reply:
Adding to common schema